web/display/minimal/index.php

```diff
@@ -34,25 +35,54 @@
 			});
 		</script>
 	</head>
+
 	<body>
 
 		<!-- minimal.html -->
-		<?php include ("gaugevalues.php"); ?>
-		<script src="display/minimal/gauge.min.js?ver=20220510"></script>
+		<?php include("gaugevalues.php"); ?>
+		<script src="display/minimal/gauge.js?ver=20230106"></script>
 		<link rel="stylesheet" href="display/minimal/minimal.css?ver=20220510">
 
 		<div id="main">
-			<div id="gaugedivlp1" style="position: absolute; top: 10px; left: 10px; bottom: 10px; right: 50%;">
-				<canvas id="lp1" style="height: 600px; top: -150px; left: 20px; position: absolute; width: 760px;"></canvas>
-				<div id="lp1t" style="font-size: 35px; top: 290px; left: 265px; text-align:center; position: absolute; width: 265px; color: white;">0</div>
-				<canvas id="lp1s" style="height: 520px; top: -80px; left: 30px; position: absolute; width: 740px;"></canvas>
-				<div id="lp1st" style="font-size: 35px; top: 230px; left: 265px; text-align:center; position: absolute; width: 265px; color: white;">0</div>
-				<div style="top: 350px; left: 265px; text-align: center; position: absolute; width: 265px; font-size: 32px;">
-					<i id="lp1plugstat" class="fas fa-plug"></i>
-					<i id="lp1disabled" class="fas fa-lock" style="color: red"></i>
-					<i id="lp1enabled" class="fas fa-unlock hide" style="color: green"></i>
+			<!-- <div id="800x480_Frame" style="position: absolute; top: 0px; left: 0px; height: 480px; width: 800px; border:1px solid white;"> </div> -->
+			<?php if ($lastmanagementold == "1\n") { ?>
```

Comment:
Nach Anpassung der gaugevalues.php:
```suggestion
			<?php if ($lastmanagementold == "1") { ?>
```

Reply:
übernommen...